hash_practice/exercises.py

```diff
@@ -2,18 +2,46 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n^2)? The fucntion only loops through the length of strings for 

```

Comment:
⏱ Sorting is an O(n) operation, however if we assume all strings are valid english words, we can say this is an O(1) operation because n will never be very large. 

Therefore you just have the for loop through the elements of `strings` which would make this O(n)